llvm/test/CodeGen/AArch64/interleaved-load-combine-pr90695.ll

```diff
@@ -0,0 +1,19 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 4
+; RUN: opt -S -passes=interleaved-load-combine < %s | FileCheck %s
+
+target triple = "aarch64-unknown-windows-gnu"
```

Comment:
never seen this triple before :)

Reply:
![image](https://github.com/llvm/llvm-project/assets/106511/d58a3c5d-1deb-42f5-be0b-3f6c9b07cbfa)

It's one of our CI machines over at the Zig project :-)